main.py

```diff
@@ -14,6 +20,11 @@ def setup(self):
         """
         log.info("SETUP Kytos/kafka_events")
 
+        self._kafka_handler: KafkaManager = KafkaDomainManager()
+        self._async_loop: AbstractEventLoop = asyncio.get_running_loop()
+
+        self._async_loop.create_task(self._kafka_handler.setup())
```

Comment:
See Python `create_task` weak ref potential issue: https://docs.python.org/3/library/asyncio-task.html#creating-tasks. This needs to hold a ref similarly to what we do on kytos core https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L417-L428.